relay-base-schema/src/metrics/mri.rs

```diff
@@ -464,6 +464,23 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_normalize_name_length() {
+        let long_mri = "c:custom/ThisIsACharacterLongStringForTestingPurposesToEnsureThatWeHaveEnoughCharactersToWorkWithAndToCheckIfOurFunctionProperlyHandlesSlicingAndNormalizationWithoutErrors";
```

Comment:
makes sense to also have a test on how it behaves when we have replacement characters in the name which get replaced with a single underscore